HouseRules_Essentials/Rules/LevelSequenceOverriddenRule.cs

```diff
@@ -0,0 +1,63 @@
+﻿namespace HouseRules.Essentials.Rules
+{
+    using System.Collections.Generic;
+    using System.Linq;
+    using Boardgame;
+    using Boardgame.LevelLoading;
+    using Boardgame.Networking;
+    using Data.GameData;
+    using DataKeys;
+    using HarmonyLib;
+    using HouseRules.Types;
+    using UnityEngine;
+
+    public sealed class LevelSequenceOverriddenRule : Rule, IConfigWritable<List<string>>, IMultiplayerSafe
+    {
+        public override string Description => "LevelSequence is overridden";
+
+        protected override SyncableTrigger ModifiedSyncables => SyncableTrigger.NewPieceModified | SyncableTrigger.StatusEffectDataModified;
+
+        private readonly List<string> _adjustments;
+        private List<string> _originals;
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="LevelSequenceOverriddenRule"/> class.
+        /// </summary>
+        /// <param name="adjustments">List of strings of LevelNames.</param>
+        public LevelSequenceOverriddenRule(List<string> adjustments)
+        {
+            _adjustments = adjustments;
+            _originals = new List<string>();
+        }
+
+        public List<string> GetConfigObject() => _adjustments;
+
+        protected override void OnPostGameCreated(GameContext gameContext)
+        {
+            _originals = ReplaceExistingProperties(_adjustments);
+        }
+
+        protected override void OnDeactivate(GameContext gameContext)
+        {
+            ReplaceExistingProperties(_originals);
+        }
+
+        /// <summary>
+        /// Replaces LevelSequence levels with predefined list.
+        /// </summary>
+        /// <returns>List of previous LevelSequnece levels that are now replaced.</returns>
+        private static List<string> ReplaceExistingProperties(List<string> replacements)
+        {
+            var levelManager = Resources.FindObjectsOfTypeAll<LevelManager>().First();
+            var levelSequence = Traverse.Create(levelManager).Field<LevelSequence>("levelSequence").Value;
```

Comment:
FYI instead of `Resources.FindObjects...`, we could find the level sequence using the `GameContext`:
```cs
Traverse.Create(_gameContext.gameStateMachine).Field<LevelSequence>("levelSequence").Value;
```

`Resources.FindObjects...` is a very expensive operation, but fortunately we've removed a lot of its use since `v1.0.0` and it's use in `OnPre/PostGameCreated` just manifests itself as a longer black loading-screen - which feels seemless anyway.

Reply:
Was unablet to get this alternative approach to work. Will merge it as-is for now.